Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt

```diff
@@ -237,6 +272,21 @@ class DartToolingDaemonService private constructor(private val project: Project)
       //  println("received response from streamListen")
       //  println(response)
       //}
+
+      // Example request to test registering a service method
```

Comment:
This part should be commented out and only used if we want to test this out

Reply:
Ok, pushed as commented out